CHANGELOG.md

```diff
@@ -38,6 +38,7 @@
 - \#2845 - Misleading error feedback in app creation modal on port conflict
 - \#2812 - Cannot change configuration of Marathon app after deployment
 - \#2821 - Tooltip content update issue
+- \#2972 - Add more inline task information
```

Comment:
Well, 0.14 broke the host+ports inline info for tasks, so I guess we could say that this is a bugfix following a change introduced in 0.14?